components/UpdatesWithData.js

```diff
@@ -77,13 +78,16 @@ class UpdatesWithData extends React.Component {
         )}
         <UpdateFilters values={query} onChange={onChange} />
         <Box mt={4} mb={5}>
-          <Updates
-            collective={collective}
-            updates={updates}
-            editable={!compact}
-            fetchMore={this.props.fetchMore}
-            LoggedInUser={LoggedInUser}
-          />
+          {data.loading && <Loading />}
```

Comment:
:ice_cream: It's purely a style preference (no requested change here), but I like to use ternary conditions for these cases. I find them very clear to read, and it prevents checking multiple time the same thing.

```jsx
{data.loading ? (
  <Loading />
) : (
  <Component />
)
```

Reply:
Thanks, let me keep this in mind. 👍🏽 I've made the change.